### ISSUE TITLE
Customized PHP Formatting

### ISSUE THREAD

**Issue by kristher1619** [marlonfan/coc-phpls#66]
Is there a way that we can customize the options when php is enabled in formatOnSaveFiletypes?

I want to load my own phpcs fixer when doing auto format

**Comment by haringsrob** [comment]
Yes, using a phpcs config for formatting would be really helpful!
